### ISSUE TITLE
Publishing on NPM

### ISSUE THREAD

**Issue by nwhite89** [InteractiveAdvertisingBureau/safeframe#11]
How do you feel about publishing this project on NPM? It be great for personal development when trying to install safeframe rather than having to download via Git / Tarballs etc.

Thanks

**Comment by seansd** [comment]
Yeah we could, although there's nothing in this repro that does anything server-side, so it would just be for packaging purposes.

I have a bunch of backlogged items that I want to get in as well, so I think I'd at least wait till I can get around to getting those checked in.

**Comment by nwhite89** [comment]
Sounds good cheers @seansd
